app/events/views.py

```diff
@@ -58,16 +65,16 @@ def create():
 		return jsonify({"message" : "you have a similar event in the same location"}), 302
 
 @events.route('/all')
-@events.route('/all/<int:page>')
-def get_all(page=1):
+def get_all():
 	"""fetch all events available"""
 	#fetch the first 15 events based on event date
-	result = Events.query.filter(cast(Events.event_date, Date) >=  date.today()).paginate(page, per_page=15, error_out=False)
+	page = request.args.get("page", default=1, type=int)
+	per_page = request.args.get("per_page", default=15, type=int)
```

Comment:
good implementation of pagination and page limiter dynamically

Reply:
no, you can only filter by one attribute at a time. though it also makes sense to use both.

Reply:
Yes it would be advisable to have both.

Reply:
just implemented that feature, check it out